lib/puppet/file_system/windows.rb

```diff
@@ -100,4 +110,18 @@ def raise_if_symlinks_unsupported
     end
   end
 
+  def glob_expands_paths
```

Comment:
Should this be `glob_expands_path?` since it seems to be a boolean query method ?